### PR TITLE
Use go.uber.org import path

### LINKED ISSUE #11: go.uber.org/atomic?
Should we move this library to a vanity import path?

### PR DESCRIPTION
**Pull request by prashantv**
Fixes #11 

Note: we should add `atomic` to the `go.uber.org` repo first.

### CODE REVIEW

**Comment by abhinav** [comment]
This will be a 2.0, right?

**Comment by prashantv** [comment]
Thinking about it, since this package doesn't import anything else, I think we can make it work as either go.uber.org or github.com/uber-go/atomic.

What do you think about that?

**Comment by abhinav** [comment]
Yeah, that would work if you remove the canonical import path comment. You
would just have to remember to bump to 2.0 if you do end up adding a
subpackage at some point in the future.

**Comment by prashantv** [comment]
Yeah, I'd like to have Travis test it under both import paths if possible.

**Comment by abhinav** [comment]
Okay, SGTM. Go for it.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: go
+go_import_path: go.uber.org/thriftrw
```
Comment (breerly) [inline .travis.yml]: yeah, this should do the trick.

File: atomic.go [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 // Package atomic provides simple wrappers around numerics to enforce atomic
 // access.
-package atomic
+package atomic // import "go.uber.org/atomic"
```
Comment (breerly) [inline atomic.go]: Are you sure you want the canonical path set? This will force users to use this import path.

**Comment by breerly** [comment]
Don't forget to update the go-doc link in the README :)

**Comment by breerly** [comment]
@abhinav let's try not to forget to update thriftrw dep to use go.uber.org/atomic before cutting the 1.0

**Comment by keegancsmith** [comment]
I realise this was closed a while ago, but interested why you chose support both import paths? According to godoc all uses of the older import path are controlled by the uber org, why not update them and add back the canonical import? https://godoc.org/github.com/uber-go/atomic?importers

**Comment by prashantv** [comment]
@keegancsmith Since this is a single package import, it works fine under either import path. Given that it works fine either way, breaking the old import without any advantages didn't seem worth it. I don't have strong opinions either way, but thought why break something without a good reason.

**Pull request merged**